src/System.Threading/ref/System.Threading.cs

```diff
@@ -324,6 +330,24 @@ public partial class SynchronizationLockException : System.Exception
         public SynchronizationLockException() { }
         public SynchronizationLockException(string message) { }
         public SynchronizationLockException(string message, System.Exception innerException) { }
+        protected SynchronizationLockException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+    }
+    public sealed partial class ThreadAbortException : System.SystemException
```

Comment:
Ok I suppose I can just implement a new type in CoreFX and leave the current one in CoreCLR for now